### PR TITLE
Languages

### PR DESCRIPTION
**Pull request by olovy**
* Add Language to domain/range of broader/narrower]
* Update Language cards
* Add/update some languages

See individual commits

### CODE REVIEW

**Review by niklasl — APPROVED** [review]
LGTM! If we have additional languages to link using `:broader` to aid in work matching, add these too. (Ping @kwahlin - I recall having seen some long-hanging fruits in a clustering example? Also @klngwll did you have any reference to LC mapping languages like this?)

File: source/languages.ttl [hunk 1]
```diff
@@ -27,7 +27,8 @@
     rdfs:comment "-1500. \"9fs\" Librisdefinierad MARC-kod."@sv ;
     skos:notation "9fs",
         "9fs"^^kbv:LibrisLocalLanguageCode ;
-    skos:prefLabel "Fornsvenska"@sv .
+    skos:prefLabel "Fornsvenska"@sv ;
+    skos:hiddenLabel "svenska (fornsvenska)" .
```
Comment (niklasl) [inline source/languages.ttl:31]: Add language tags on (all) the hidden ones too?

Comment (olovy) [reply source/languages.ttl:31]: I think we could do it for the ones where it is obvious/unambigous.
Need to add `hiddenLabelByLang` to context though.

**Comment by klngwll** [comment]
> LGTM! If we have additional languages to link using `:broader` to aid in work matching, add these too. (Ping @kwahlin - I recall having seen some long-hanging fruits in a clustering example? Also @klngwll did you have any reference to LC mapping languages like this?)

Looking into it now only was available as narrower terms on language groups like https://id.loc.gov/vocabulary/iso639-5/ine.html and exact matches to their subject headings from the 639-2 langs. So we can leave LC references for now.

File: source/languages.ttl (hunk 1 above)
Comment (niklasl) [reply source/languages.ttl]: Added that and some obvious tagged hidden values.

**Pull request merged**